InnerEye/ML/reports/classification_crossval_report.ipynb

```diff
@@ -0,0 +1,272 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%javascript\n",
+    "IPython.OutputArea.prototype._should_scroll = function(lines) {\n",
+    "    return false;\n",
+    "}\n",
+    "// Stops auto-scrolling so entire output is visible: see https://stackoverflow.com/a/41646403"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2",
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Default parameter values. They will be overwritten by papermill notebook parameters.\n",
+    "# This cell must carry the tag \"parameters\" in its metadata.\n",
+    "from pathlib import Path\n",
+    "import pickle\n",
+    "import codecs\n",
+    "\n",
+    "innereye_path = Path.cwd().parent.parent.parent\n",
+    "train_metrics_csv = \"\"\n",
+    "val_metrics_csv = innereye_path / 'Tests' / 'ML' / 'reports' / 'val_metrics_classification.csv'\n",
+    "test_metrics_csv = innereye_path / 'Tests' / 'ML' / 'reports' / 'test_metrics_classification.csv'\n",
+    "number_best_and_worst_performing = 20\n",
+    "config = \"\"\n",
+    "is_crossval_report = False"
```

Comment:
what do you mean? we pass in `"is_crossval_report": False` for [standard reports](https://github.com/microsoft/InnerEye-DeepLearning/blob/2c296c7ae4e5cab151315ca41d41c43c237c2163/InnerEye/ML/reports/notebook_report.py#L156) and `True` for [cross-validation](https://github.com/microsoft/InnerEye-DeepLearning/blob/2c296c7ae4e5cab151315ca41d41c43c237c2163/InnerEye/ML/reports/notebook_report.py#L179)

Reply:
But it is two different notebooks so this param does not need to be passed to the notebook at all. If it is standard report you will use another ipynb file no?

Reply:
now we use `classification_crossval_report.ipynb` as the only template for both standard and crossval reports, and this switch controls what gets displayed. should I delete the legacy template file `classification_report.ipynb` to avoid confusion?